lighthouse-core/lib/sentry.js

```diff
@@ -20,6 +20,11 @@ const sentryApi = {
   getContext: noop,
 };
 
+const IGNORED_ERRORS = [
```

Comment:
afaict these two aren't even being reported to sentry. why start excluding them?

Reply:
I marked the unable to load the page ones as ignore in sentry, is there a reason you want to track them?

Reply:
ah  unable to load is ignored.  

but tracingalreadystarted?  is that even reported?  getting a repro via CLI would be useful as it seems the only repros are with devtools/ extension

Reply:
alright, fair enough

just seemed like information we wouldn't actually learn anything from as we have no reason to believe it has anything to do with the URL being audited